ansible/module_utils/port_utils.py

```diff
@@ -471,6 +471,9 @@ def get_port_alias_to_name_map(hwsku, asic_name=None):
             port_alias_to_name_map['etp65'] = "Ethernet512"
             if hwsku == "Mellanox-SN5610N-C224O8":
                 port_alias_to_name_map['etp66'] = "Ethernet520"
+        elif hwsku == "ACS-SN4280":
```

Comment:
@zjswhhh do you mind to help check the HWSKU and confirm it indeed starts with ACS not Mellanox?

Reply:
Yes it's `HwSKU: ACS-SN4280`.

But I didn't need this change to generate config or to add-topo...